File: 2order-frontend/src/model/Porudzbina.ts
import Korisnik from "./Korisnik";
import Racun from "./Racun";
import StavkaMenija from "./StavkaMenija";

interface Porudzbina {
    id?: number;
    korisnikId: number;
    korisnik?: Korisnik;
    stavkaMenijaId: number;
    stavkaMenija?: StavkaMenija;
    racunId?: number;
    racun?: Racun;
}

export default Porudzbina;

File: 2Order/2Order.Domain/Model/Porudzbina.cs
using System.Text.Json.Serialization;
using System;
using System.Collections.Generic;
using System.Text;

namespace _2Order.Domain.Model
{
    public class Porudzbina
    {
        public int Id { get; set; }
        public int KorisnikId { get; set; }
        public Korisnik Korisnik { get; set; }
        public int StavkaMenijaId { get; set; }
        public StavkaMenija StavkaMenija { get; set; }
        public int RacunId { get; set; }
        [JsonIgnore]
        public Racun Racun { get; set; }
        public string Sifra { get; set; }
    }
}
